SemanticUI/Modules/Dropdown/Select.elm

```diff
@@ -0,0 +1,327 @@
+module SemanticUI.Modules.Dropdown.Select exposing
+    ( Builder
+    , Config
+    , Select(..)
+    , Variation(..)
+    , attributes
+    , button
+    , caret
+    , disabled
+    , fluid
+    , inline
+    , linkItem
+    , scrolling
+    , select
+    , toCustomHtml
+    , toHtml
+    , toItem
+    , toggleEvent
+    )
+
+{-| A [dropdown](https://semantic-ui.com/modules/dropdown.html) with selectable options in its drawer.
+
+Example of `Select.select` :
+
+    Select.select
+        { identifier = "nav-select"
+        , onToggle = ToggleNavSelect
+        , onSelect = identity
+        , drawerState = model.navDrawerState
+        }
+        |> Select.toHtml
+            { options = [ NavHome, NavBack ]
+            , optionLabel = text << navToString
+            }
+
+See `SematicUI.Modules.Dropdown.Selection` for dropdowns that indicate a current selection state.
+
+-}
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Builder as Html
+import Html.Events exposing (..)
+import SemanticUI.Elements.Button as Button
+import SemanticUI.Modules.Dropdown as Dropdown exposing (Dropdown(..))
+import SemanticUI.Modules.Dropdown.Drawer as Drawer
+import SemanticUI.Modules.Dropdown.Toggle as Toggle
+
+
+{-| Select variations. For internal use only.
```

Comment:
If it's for internal use, why is it exported?

Reply:
Good point... I think this comment should probably read similar to the comment above `Config` instead of this.

The export is an escape hatch in case people want to just deal directly with the config rather than the carefully thought out constructors defined here (`inline` / `button` / etc).